locale/zh_CN/LC_MESSAGES/tutorial.po

```diff
@@ -39,6 +42,9 @@ msgid ""
 "Fortran standard library ([stdlib]) as dependency. In the package "
 "manifest we define *stdlib* in the *dependencies* table:"
 msgstr ""
+"我们从一个使用fpm的新项目开始，我们想构建一个命令行应用程序来读取文件，找到某个模式（pattern）并替换它。"
+"由于我们不想自己编写替换函数，我们将使用Fortran标准库（[stdlib](https://github.com/fortran-lang/stdlib)）作为*依赖项*。"
```

Comment:
> I found that having the links declared separately, rather than inline is somewhat easier when translating, because we don't have to repeat the URL.

@awvwgk , It may be that I don't know how to achieve it. I tried not to add duplicate URLs in the translation, but it seems that the generated effect is not as expected, looks like:

```md
"由于我们不想自己编写替换函数，我们将使用Fortran标准库（[stdlib]）作为*依赖项*。"
```

![image](https://user-images.githubusercontent.com/32035096/149313730-8713a6c4-301c-41a4-98cd-077cff785c7c.png)

Reply:
You are right, that doesn't work as expected in the translations. I changed it to the inline link.

Reply:
LGTM~